src/server/coprocessor/endpoint.rs

```diff
@@ -174,6 +175,22 @@ impl RequestTask {
                   self.req.get_ranges().get(0));
         }
     }
+
+    pub fn parse_sel(&mut self) -> Result<SelectRequest> {
+        let tp = self.req.get_tp();
+        match tp {
+            REQ_TYPE_SELECT | REQ_TYPE_INDEX => {
+                let mut sel = SelectRequest::new();
+                if let Err(e) = sel.merge_from_bytes(self.req.get_data()) {
+                    return Err(box_err!(e));
+                }
+                let start_ts = sel.get_start_ts();
+                self.start_ts = Some(start_ts);
+                Ok(sel)
+            }
+            _ => Err(box_err!("unsupported tp {}", tp)),
```

Comment:
This is the original logic. And I would like to retain it in case if it was used as the monitoring key word online. @andelf